.github/workflows/bytebase-sql-review.yml

```diff
@@ -0,0 +1,42 @@
+on: [pull_request]
+jobs:
+  bytebase-sql-review:
+    runs-on: ubuntu-latest
+    name: SQL Review
+    steps:
+      - name: SQL advise
+        run: |
+          API="https://bytebase.inspection.alpha.canada.ca/hook/sql-review/c033cbf2-f0e6-4c1a-89ba-ced209b06fba-1712080200"
```

Comment:
this could/should also be a secret?

Reply:
also naming BYTEBASE_API and BYTEBASE_TOKEN would make it more obvious what these constants are.

Reply:
done in reusable workflow